src/main/java/spoon/metamodel/MMMethodKind.java

```diff
@@ -77,13 +77,12 @@
 	private final Predicate<CtMethod<?>> detector;
 	private final int level;
 	private final boolean multi;
-	private final int valueParameterIndex;
 
 	MMMethodKind(int valueParameterIndex, boolean multi, int level, Predicate<CtMethod<?>> detector) {
 		this.multi = multi;
 		this.level = level;
 		this.detector = detector;
-		this.valueParameterIndex = valueParameterIndex;
+		// valueParameterindex is never used
```

Comment:
Instead of commenting the fields and co, it would be better to simply remove them

Reply:
I was unsure about it, because the field is marked for future use.

Reply:
let's remove it completely

Reply:
You are correct, it's an Enum => we can simply remove the parameter because enums cant be subclassed and the value was private.